pkg/mediationcontainer/client_websocket_transport_test.go

```diff
@@ -72,6 +76,10 @@ func TestCreateClientWebSocketTransport(t *testing.T) {
 					LocalAddress: item.localAddress,
 				},
 				item.bindingChannel,
+				SdkProtocolConfig{
+					RegistrationTimeoutSec: 30,
+					ExitOnProtocolTimeout:  false,
```

Comment:
Perhaps rename to DeleteKubeturboPODOnProtocolTimeout or DeletePODOnProtocolTimeout

Reply:
It won't be accurate to say delete pod, as `os.Exit()` only terminates the running container, and k8s will restart the container, the pod is not deleted (i.e., it will still have the same pod ID)

Reply:
Changed to `ExitProbePodOnProtocolTimeout`